src/components/basic-element/types/textedit.js

```diff
@@ -159,7 +158,7 @@ module.exports = {
       if(!this.state.editing) {
         this.onTextUpdate({
           target: {
-            value: this.props.innerHTML.trim()
```

Comment:
Were the changes other than this one required to retain the leading and trailing spaces? Or were you just cleaning things up?